### ISSUE TITLE
Implement "New Run" view on Agent Runs page

### ISSUE THREAD

**Issue by Pwuts** [Significant-Gravitas/AutoGPT#8783]
* Part of #8780

![Screenshot of the "New Run" view on the Agent Runs page](https://uploads.linear.app/a47946b5-12cd-4b3d-8822-df04c855879f/aad079b8-03eb-4218-9a4b-6ab7c049ed4a/75a5c4b1-7c27-4df7-9702-10142548e08a?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2E0Nzk0NmI1LTEyY2QtNGIzZC04ODIyLWRmMDRjODU1ODc5Zi9hYWQwNzliOC0wM2ViLTQyMTgtOWE0Yi02YWI3YzA0OWVkNGEvNzVhNWM0YjEtN2MyNy00ZGY3LTk3MDItMTAxNDI1NDhlMDhhIiwiaWF0IjoxNzMyNjQ1MTE3LCJleHAiOjMzMzAzMjA1MTE3fQ.xajyLjfqKDlyYxnpGWwKp9z-mQ-OEuymqY3YUm8JF_M)

### Highlights & notes

* Basically ported from the "Run" dialog that's currently in the Builder
* **Note:** in the wireframes, the title in this view is "(New) {agent_name}", which I find confusing so we can omit this pending further refinement of the design
* "Schedule run" dialog
  * Basically ported from the "Schedule" dialog that's currently in the Monitor page
  * **Note:** in the current design, the "set up schedule" input is in the form rather than in the action column. I've proposed to changed this in a comment, but am waiting on a response from Olivia.
